// @strapi/admin/build/2077.058590f4.chunk.js
(self.webpackChunk_strapi_admin=self.webpackChunk_strapi_admin||[]).push([[2077],{74201:(O,k,g)=>{"use strict";O.exports=g(45214)},45214:function(O,k,g){(function(x,v){O.exports=v(g(32735),g(19615),g(103))})(this,function(x,v,m){return function(t){var u={};function n(r){if(u[r])return u[r].exports;var i=u[r]={i:r,l:!1,exports:{}};return t[r].call(i.exports,i,i.exports,n),i.l=!0,i.exports}return n.m=t,n.c=u,n.d=function(r,i,d){n.o(r,i)||Object.defineProperty(r,i,{enumerable:!0,get:d})},n.r=function(r){typeof Symbol!="undefined"&&Symbol.toStringTag&&Object.defineProperty(r,Symbol.toStringTag,{value:"Module"}),Object.defineProperty(r,"__esModule",{value:!0})},n.t=function(r,i){if(1&i&&(r=n(r)),8&i||4&i&&typeof r=="object"&&r&&r.__esModule)return r;var d=Object.create(null);if(n.r(d),Object.defineProperty(d,"default",{enumerable:!0,value:r}),2&i&&typeof r!="string")for(var s in r)n.d(d,s,function(f){return r[f]}.bind(null,s));return d},n.n=function(r){var i=r&&r.__esModule?function(){return r.default}:function(){return r};return n.d(i,"a",i),i},n.o=function(r,i){return Object.prototype.hasOwnProperty.call(r,i)},n.p="",n(n.s=66)}([function(t,u,n){t.exports=n(21)()},function(t,u){t.exports=x},function(t,u){t.exports=v},function(t,u){t.exports=function(n,r){return r||(r=n.slice(0)),Object.freeze(Object.defineProperties(n,{raw:{value:Object.freeze(r)}}))},t.exports.default=t.exports,t.exports.__esModule=!0},function(t,u,n){var r=n(26);t.exports=function(i,d){if(i==null)return{};var s,f,c=r(i,d);if(Object.getOwnPropertySymbols){var p=Object.getOwnPropertySymbols(i);for(f=0;f<p.length;f++)s=p[f],d.indexOf(s)>=0||Object.prototype.propertyIsEnumerable.call(i,s)&&(c[s]=i[s])}return c},t.exports.default=t.exports,t.exports.__esModule=!0},function(t,u){function n(){return t.exports=n=Object.assign||function(r){for(var i=1;i<arguments.length;i++){var d=arguments[i];for(var s in d)Object.prototype.hasOwnProperty.call(d,s)&&(r[s]=d[s])}return r},t.exports.default=t.exports,t.exports.__esModule=!0,n.apply(this,arguments)}t.exports=n,t.exports.default=t.exports,t.exports.__esModule=!0},function(t,u,n){"use strict";n.r(u),n.d(u,"Box",function(){return z});var r,i=n(3),d=n.n(i),s=n(2),f=n.n(s),c=n(7),p=n(1),y=n.n(p),S=n(0),a=n.n(S),h=function(e){return y.a.createElement("div",e)},w={background:void 0,borderColor:void 0,color:void 0,hiddenS:!1,hiddenXS:!1,padding:void 0,paddingTop:void 0,paddingRight:void 0,paddingBottom:void 0,paddingLeft:void 0,hasRadius:!1,shadow:void 0,children:null,shrink:void 0,grow:void 0,basis:void 0,flex:void 0,_hover:function(){}},b={_hover:a.a.func,background:a.a.string,basis:a.a.oneOfType([a.a.string,a.a.string]),borderColor:a.a.string,children:a.a.oneOfType([a.a.node,a.a.string]),color:a.a.string,flex:a.a.oneOfType([a.a.string,a.a.string]),grow:a.a.oneOfType([a.a.string,a.a.string]),hasRadius:a.a.bool,hiddenS:a.a.bool,hiddenXS:a.a.bool,padding:a.a.oneOfType([a.a.number,a.a.arrayOf(a.a.number)]),paddingBottom:a.a.oneOfType([a.a.number,a.a.arrayOf(a.a.number)]),paddingLeft:a.a.oneOfType([a.a.number,a.a.arrayOf(a.a.number)]),paddingRight:a.a.oneOfType([a.a.number,a.a.arrayOf(a.a.number)]),paddingTop:a.a.oneOfType([a.a.number,a.a.arrayOf(a.a.number)]),shadow:a.a.string,shrink:a.a.oneOfType([a.a.string,a.a.string])};h.defaultProps=w,h.propTypes=b;var T={color:!0},z=f.a.div.withConfig({shouldForwardProp:function(e,o){return!T[e]&&o(e)}})(r||(r=d()([`
  // Font
  font-size: `,`;

  // Colors
  background: `,`;
  color: `,`;

  // Spaces
  `,`
  `,`
  `,`
  `,`
  `,`
  `,`
  `,`
  `,`
  `,`

  // Responsive hiding
  `,`
  `,`
  

  // Borders
  border-radius: `,`;
  border-style: `,`;
  border-width: `,`;
  border-color: `,`;
  border: `,`;

  // Shadows
  box-shadow: `,`;

  // Handlers
  pointer-events: `,`;
  &:hover {
    `,`
  }

  // Display
  display: `,`;

  // Position
  position: `,`;
  left: `,`;
  right: `,`;
  top: `,`;
  bottom: `,`;
  z-index: `,`;
  overflow: `,`;
  cursor: `,`;

  // Size
  width: `,`;
  max-width: `,`;
  min-width: `,`;
  height: `,`;
  max-height: `,`;
  min-height: `,`;

  // Animation
  transition: `,`;
  transform: `,`;
  animation: `,`;

  //Flexbox children props
  flex-shrink: `,`;
  flex-grow: `,`;
  flex-basis: `,`;
  flex: `,`;

  // Text
  text-align: `,`;
  text-transform: `,`;
  line-height: `,`;

  // Cursor
  cursor: `,`;
`])),function(e){var o=e.fontSize;return e.theme.fontSizes[o]||o},function(e){var o=e.theme,l=e.background;return o.colors[l]},function(e){var o=e.theme,l=e.color;return o.colors[l]},function(e){var o=e.theme,l=e.padding;return Object(c.a)("padding",l,o)},function(e){var o=e.theme,l=e.paddingTop;return Object(c.a)("padding-top",l,o)},function(e){var o=e.theme,l=e.paddingRight;return Object(c.a)("padding-right",l,o)},function(e){var o=e.theme,l=e.paddingBottom;return Object(c.a)("padding-bottom",l,o)},function(e){var o=e.theme,l=e.paddingLeft;return Object(c.a)("padding-left",l,o)},function(e){var o=e.theme,l=e.marginLeft;return Object(c.a)("margin-left",l,o)},function(e){var o=e.theme,l=e.marginRight;return Object(c.a)("margin-right",l,o)},function(e){var o=e.theme,l=e.marginTop;return Object(c.a)("margin-top",l,o)},function(e){var o=e.theme,l=e.marginBottom;return Object(c.a)("margin-bottom",l,o)},function(e){var o=e.theme;return e.hiddenS?"".concat(o.mediaQueries.tablet," { display: none; }"):void 0},function(e){var o=e.theme;return e.hiddenXS?"".concat(o.mediaQueries.mobile," { display: none; }"):void 0},function(e){var o=e.theme,l=e.hasRadius,j=e.borderRadius;return l?o.borderRadius:j},function(e){return e.borderStyle},function(e){return e.borderWidth},function(e){var o=e.borderColor;return e.theme.colors[o]},function(e){var o=e.theme,l=e.borderColor,j=e.borderStyle,M=e.borderWidth;if(l&&!j&&!M)return"1px solid ".concat(o.colors[l])},function(e){var o=e.theme,l=e.shadow;return o.shadows[l]},function(e){return e.pointerEvents},function(e){var o=e._hover,l=e.theme;return o?o(l):void 0},function(e){return e.display},function(e){return e.position},function(e){var o=e.left;return e.theme.spaces[o]||o},function(e){var o=e.right;return e.theme.spaces[o]||o},function(e){var o=e.top;return e.theme.spaces[o]||o},function(e){var o=e.bottom;return e.theme.spaces[o]||o},function(e){return e.zIndex},function(e){return e.overflow},function(e){return e.cursor},function(e){var o=e.width;return e.theme.spaces[o]||o},function(e){var o=e.maxWidth;return e.theme.spaces[o]||o},function(e){var o=e.minWidth;return e.theme.spaces[o]||o},function(e){var o=e.height;return e.theme.spaces[o]||o},function(e){var o=e.maxHeight;return e.theme.spaces[o]||o},function(e){var o=e.minHeight;return e.theme.spaces[o]||o},function(e){return e.transition},function(e){return e.transform},function(e){return e.animation},function(e){return e.shrink},function(e){return e.grow},function(e){return e.basis},function(e){return e.flex},function(e){return e.textAlign},function(e){return e.textTransform},function(e){return e.lineHeight},function(e){return e.cursor});z.defaultProps=w,z.propTypes=b},function(t,u,n){"use strict";var r=n(11),i=n.n(r),d=n(15),s=n.n(d);u.a=function(f,c,p){var y=c;if(Array.isArray(c)||s()(c)!=="object"||(y=[c==null?void 0:c.desktop,c==null?void 0:c.tablet,c==null?void 0:c.mobile]),y!==void 0){if(Array.isArray(y)){var S=y,a=i()(S,3),h=a[0],w=a[1],b=a[2],T="".concat(f,": ").concat(p.spaces[h],";");return w!==void 0&&(T+="".concat(p.mediaQueries.tablet,`{
          `).concat(f,": ").concat(p.spaces[w],`;
        }`)),b!==void 0&&(T+="".concat(p.mediaQueries.mobile,`{
          `).concat(f,": ").concat(p.spaces[b],`;
        }`)),T}var z=p.spaces[y]||y;return"".concat(f,": ").concat(z,";")}}},function(t,u,n){"use strict";n.d(u,"a",function(){return r}),n.d(u,"b",function(){return i}),n.d(u,"c",function(){return d}),n.d(u,"d",function(){return s}),n.d(u,"e",function(){return f}),n.d(u,"f",function(){return c}),n.d(u,"g",function(){return p}),n.d(u,"h",function(){return y});var r="alpha",i="beta",d="delta",s="epsilon",f="omega",c="pi",p="sigma",y=[r,i,d,s,f,c,p]},function(t,u,n){"use strict";n.r(u),n.d(u,"Typography",function(){return e});var r,i=n(3),d=n.n(i),s=n(2),f=n.n(s),c=n(13),p=n(1),y=n.n(p),S=n(0),a=n.n(S),h=n(8),w=function(o){return y.a.createElement("div",o)},b={ellipsis:!1,fontWeight:void 0,fontSize:void 0,lineHeight:void 0,textColor:void 0,textAlign:void 0,textTransform:void 0,variant:h.e},T={ellipsis:a.a.bool,fontSize:a.a.oneOfType([a.a.number,a.a.string]),fontWeight:a.a.string,lineHeight:a.a.oneOfType([a.a.number,a.a.string]),textAlign:a.a.string,textColor:a.a.string,textTransform:a.a.string,variant:a.a.oneOf(h.h)};w.defaultProps=b,w.propTypes=T;var z={fontSize:!0,fontWeight:!0},e=f.a.span.withConfig({shouldForwardProp:function(o,l){return!z[o]&&l(o)}})(r||(r=d()([`
  font-weight: `,`;
  font-size: `,`;
  line-height: `,`;
  color: `,`;
  text-align: `,`;
  text-transform: `,`;
  `,`
  `,`
`])),function(o){var l=o.theme,j=o.fontWeight;return l.fontWeights[j]},function(o){var l=o.theme,j=o.fontSize;return l.fontSizes[j]},function(o){var l=o.theme,j=o.lineHeight;return l.lineHeights[j]},c.b,function(o){return o.textAlign},function(o){return o.textTransform},c.a,c.c);e.defaultProps=b,e.propTypes=T},function(t,u,n){"use strict";n.r(u),n.d(u,"Flex",function(){return e});var r,i=n(3),d=n.n(i),s=n(2),f=n.n(s),c=n(6),p=n(7),y=n(1),S=n.n(y),a=n(0),h=n.n(a),w=function(o){return S.a.createElement("div",o)},b={alignItems:"center",basis:void 0,direction:"row",gap:void 0,inline:!1,justifyContent:void 0,reverse:!1,wrap:void 0},T={alignItems:h.a.string,basis:h.a.oneOfType([h.a.string,h.a.number]),direction:h.a.string,gap:h.a.oneOfType([h.a.shape({desktop:h.a.number,mobile:h.a.number,tablet:h.a.number}),h.a.number,h.a.arrayOf(h.a.number),h.a.string]),inline:h.a.bool,justifyContent:h.a.string,reverse:h.a.bool,shrink:h.a.number,wrap:h.a.string};w.defaultProps=b,w.propTypes=T;var z={direction:!0},e=f()(c.Box).withConfig({shouldForwardProp:function(o,l){return!z[o]&&l(o)}})(r||(r=d()([`
  align-items: `,`;
  display: `,`;
  flex-direction: `,`;
  flex-shrink: `,`;
  flex-wrap: `,`;
  `,`};
  justify-content: `,`;
`])),function(o){return o.alignItems},function(o){return o.inline?"inline-flex":"flex"},function(o){return o.direction},function(o){return o.shrink},function(o){return o.wrap},function(o){var l=o.gap,j=o.theme;return Object(p.a)("gap",l,j)},function(o){return o.justifyContent});e.defaultProps=b,e.propTypes=T},function(t,u,n){var r=n(27),i=n(28),d=n(24),s=n(29);t.exports=function(f,c){return r(f)||i(f,c)||d(f,c)||s()},t.exports.default=t.exports,t.exports.__esModule=!0},,function(t,u,n){"use strict";n.d(u,"a",function(){return i}),n.d(u,"c",function(){return d}),n.d(u,"b",function(){return s});var r=n(8),i=function(f){return f.ellipsis&&`
    display: block;
    white-space: nowrap;
    overflow: hidden;
    text-overflow: ellipsis;
  `},d=function(f){var c=f.variant,p=f.theme;switch(c){case r.a:return`
        font-weight: `.concat(p.fontWeights.bold,`;
        font-size: `).concat(p.fontSizes[5],`;
        line-height: `).concat(p.lineHeights[2],`;
      `);case r.b:return`
        font-weight: `.concat(p.fontWeights.bold,`;
        font-size: `).concat(p.fontSizes[4],`;
        line-height: `).concat(p.lineHeights[1],`;
      `);case r.c:return`
        font-weight: `.concat(p.fontWeights.semiBold,`;
        font-size: `).concat(p.fontSizes[3],`;
        line-height: `).concat(p.lineHeights[2],`;
      `);case r.d:return`
        font-size: `.concat(p.fontSizes[3],`;
        line-height: `).concat(p.lineHeights[6],`;
      `);case r.e:return`
        font-size: `.concat(p.fontSizes[2],`;
        line-height: `).concat(p.lineHeights[4],`;
      `);case r.f:return`
        font-size: `.concat(p.fontSizes[1],`;
        line-height: `).concat(p.lineHeights[3],`;
      `);case r.g:return`
        font-weight: `.concat(p.fontWeights.bold,`;
        font-size: `).concat(p.fontSizes[0],`;
        line-height: `).concat(p.lineHeights[5],`;
        text-transform: uppercase;
      `);default:return`
        font-size: `.concat(p.fontSizes[2],`;
      `)}},s=function(f){var c=f.theme,p=f.textColor;return c.colors[p||"neutral800"]}},,function(t,u){function n(r){return typeof Symbol=="function"&&typeof Symbol.iterator=="symbol"?(t.exports=n=function(i){return typeof i},t.exports.default=t.exports,t.exports.__esModule=!0):(t.exports=n=function(i){return i&&typeof Symbol=="function"&&i.constructor===Symbol&&i!==Symbol.prototype?"symbol":typeof i},t.exports.default=t.exports,t.exports.__esModule=!0),n(r)}t.exports=n,t.exports.default=t.exports,t.exports.__esModule=!0},,,function(t,u,n){"use strict";n.d(u,"b",function(){return r}),n.d(u,"c",function(){return i}),n.d(u,"a",function(){return d});var r=function(s){return function(f){var c=f.theme,p=f.size;return c.sizes[s][p]}},i=function(){var s=arguments.length>0&&arguments[0]!==void 0?arguments[0]:"&";return function(f){var c=f.theme,p=f.hasError;return`
      outline: none;
      box-shadow: 0;
      transition-property: border-color, box-shadow, fill;
      transition-duration: 0.2s;

      `.concat(s,`:focus-within {
        border: 1px solid `).concat(p?c.colors.danger600:c.colors.primary600,`;
        box-shadow: `).concat(p?c.colors.danger600:c.colors.primary600,` 0px 0px 0px 2px;
      }
    `)}},d=function(s){var f=s.theme;return`
  position: relative;
  outline: none;
  
  &:after {
    transition-property: all;
    transition-duration: 0.2s;
    border-radius: 8px;
    content: '';
    position: absolute;
    top: -4px;
    bottom: -4px;
    left: -4px;
    right: -4px;
    border: 2px solid transparent;
  }

  &:focus-visible {
    outline: none;
    &:after {
      border-radius: 8px;
      content: '';
      position: absolute;
      top: -5px;
      bottom: -5px;
      left: -5px;
      right: -5px;
      border: 2px solid `.concat(f.colors.primary600,`;
    }
  }
`)}},,,function(t,u,n){"use strict";var r=n(22);function i(){}function d(){}d.resetWarningCache=i,t.exports=function(){function s(p,y,S,a,h,w){if(w!==r){var b=new Error("Calling PropTypes validators directly is not supported by the `prop-types` package. Use PropTypes.checkPropTypes() to call them. Read more at http://fb.me/use-check-prop-types");throw b.name="Invariant Violation",b}}function f(){return s}s.isRequired=s;var c={array:s,bool:s,func:s,number:s,object:s,string:s,symbol:s,any:s,arrayOf:f,element:s,elementType:s,instanceOf:f,node:s,objectOf:f,oneOf:f,oneOfType:f,shape:f,exact:f,checkPropTypes:d,resetWarningCache:i};return c.PropTypes=c,c}},function(t,u,n){"use strict";t.exports="SECRET_DO_NOT_PASS_THIS_OR_YOU_WILL_BE_FIRED"},function(t,u){t.exports=function(n,r){(r==null||r>n.length)&&(r=n.length);for(var i=0,d=new Array(r);i<r;i++)d[i]=n[i];return d},t.exports.default=t.exports,t.exports.__esModule=!0},function(t,u,n){var r=n(23);t.exports=function(i,d){if(i){if(typeof i=="string")return r(i,d);var s=Object.prototype.toString.call(i).slice(8,-1);return s==="Object"&&i.constructor&&(s=i.constructor.name),s==="Map"||s==="Set"?Array.from(i):s==="Arguments"||/^(?:Ui|I)nt(?:8|16|32)(?:Clamped)?Array$/.test(s)?r(i,d):void 0}},t.exports.default=t.exports,t.exports.__esModule=!0},,function(t,u){t.exports=function(n,r){if(n==null)return{};var i,d,s={},f=Object.keys(n);for(d=0;d<f.length;d++)i=f[d],r.indexOf(i)>=0||(s[i]=n[i]);return s},t.exports.default=t.exports,t.exports.__esModule=!0},function(t,u){t.exports=function(n){if(Array.isArray(n))return n},t.exports.default=t.exports,t.exports.__esModule=!0},function(t,u){t.exports=function(n,r){var i=n==null?null:typeof Symbol!="undefined"&&n[Symbol.iterator]||n["@@iterator"];if(i!=null){var d,s,f=[],c=!0,p=!1;try{for(i=i.call(n);!(c=(d=i.next()).done)&&(f.push(d.value),!r||f.length!==r);c=!0);}catch(y){p=!0,s=y}finally{try{c||i.return==null||i.return()}finally{if(p)throw s}}return f}},t.exports.default=t.exports,t.exports.__esModule=!0},function(t,u){t.exports=function(){throw new TypeError(`Invalid attempt to destructure non-iterable instance.
In order to be iterable, non-array objects must have a [Symbol.iterator]() method.`)},t.exports.default=t.exports,t.exports.__esModule=!0},,,,,,,,,,,,,,,,,,,,,,,,function(t,u){t.exports=m},,,,,,,,,,,,,function(t,u,n){"use strict";n.r(u),n.d(u,"TextButton",function(){return P});var r,i,d,s=n(5),f=n.n(s),c=n(4),p=n.n(c),y=n(3),S=n.n(y),a=n(1),h=n.n(a),w=n(0),b=n.n(w),T=n(2),z=n.n(T),e=n(53),o=n.n(e),l=n(6),j=n(9),M=n(10),_=n(18),F=["children","startIcon","endIcon","onClick","disabled","loading"],L=Object(T.keyframes)(r||(r=S()([`
  from {
    transform: rotate(0deg);
  }
  to {
    transform: rotate(359deg);
  }
`]))),Q=z.a.div(i||(i=S()([`
  animation: `,` 2s infinite linear;
  will-change: transform;
`])),L),U=z()(M.Flex)(d||(d=S()([`
  background: transparent;
  border: none;

  &[aria-disabled='true'] {
    pointer-events: none;
    svg path {
      fill: `,`;
    }
  }

  svg {
    display: flex;
    font-size: `,`rem;
  }

  svg path {
    fill: `,`;
  }

  `,`
`])),function(C){return C.theme.colors.neutral600},.625,function(C){return C.theme.colors.primary600},_.a),P=h.a.forwardRef(function(C,X){var I=C.children,E=C.startIcon,B=C.endIcon,H=C.onClick,W=C.disabled,A=C.loading,G=p()(C,F),V=H&&!W?H:void 0,R=W||A;return h.a.createElement(U,f()({ref:X,"aria-disabled":R,onClick:V,as:"button",type:"button"},G),(E||A)&&h.a.createElement(l.Box,{as:"span",paddingRight:2,"aria-hidden":!0},A?h.a.createElement(Q,null,h.a.createElement(o.a,null)):E),h.a.createElement(j.Typography,{variant:"pi",textColor:R?"neutral600":"primary600"},I),B&&h.a.createElement(l.Box,{as:"span",paddingLeft:2,"aria-hidden":!0},B))});P.displayName="TextButton",P.defaultProps={disabled:!1,startIcon:void 0,endIcon:void 0,loading:!1,onClick:void 0},P.propTypes={children:b.a.node.isRequired,disabled:b.a.bool,endIcon:b.a.element,loading:b.a.bool,onClick:b.a.func,startIcon:b.a.element}}])})},56671:(O,k,g)=>{var x=g(35897),v=g(74741),m=v(x);O.exports=m},52920:(O,k,g)=>{var x=g(56671),v=g(81580);function m(t,u){var n=-1,r=v(t)?Array(t.length):[];return x(t,function(i,d,s){r[++n]=u(i,d,s)}),r}O.exports=m},50690:(O,k,g)=>{var x=g(26460),v=g(80166),m=g(88904),t=g(52920),u=g(36024),n=g(76535),r=g(1397),i=g(15362),d=g(77236);function s(f,c,p){c.length?c=x(c,function(a){return d(a)?function(h){return v(h,a.length===1?a[0]:a)}:a}):c=[i];var y=-1;c=x(c,n(m));var S=t(f,function(a,h,w){var b=x(c,function(T){return T(a)});return{criteria:b,index:++y,value:a}});return u(S,function(a,h){return r(a,h,p)})}O.exports=s},36024:O=>{function k(g,x){var v=g.length;for(g.sort(x);v--;)g[v]=g[v].value;return g}O.exports=k},40709:(O,k,g)=>{var x=g(22945);function v(m,t){if(m!==t){var u=m!==void 0,n=m===null,r=m===m,i=x(m),d=t!==void 0,s=t===null,f=t===t,c=x(t);if(!s&&!c&&!i&&m>t||i&&d&&f&&!s&&!c||n&&d&&f||!u&&f||!r)return 1;if(!n&&!i&&!c&&m<t||c&&u&&r&&!n&&!i||s&&u&&r||!d&&r||!f)return-1}return 0}O.exports=v},1397:(O,k,g)=>{var x=g(40709);function v(m,t,u){for(var n=-1,r=m.criteria,i=t.criteria,d=r.length,s=u.length;++n<d;){var f=x(r[n],i[n]);if(f){if(n>=s)return f;var c=u[n];return f*(c=="desc"?-1:1)}}return m.index-t.index}O.exports=v},74741:(O,k,g)=>{var x=g(81580);function v(m,t){return function(u,n){if(u==null)return u;if(!x(u))return m(u,n);for(var r=u.length,i=t?r:-1,d=Object(u);(t?i--:++i<r)&&n(d[i],i,d)!==!1;);return u}}O.exports=v},98978:(O,k,g)=>{var x=g(40314),v=g(50690),m=g(94569),t=g(63108),u=m(function(n,r){if(n==null)return[];var i=r.length;return i>1&&t(n,r[0],r[1])?r=[]:i>2&&t(r[0],r[1],r[2])&&(r=[r[0]]),v(n,x(r,1),[])});O.exports=u}}]);
